src/OfficeToolPlus/Dictionaries/Languages/ko-kr.xaml

```diff
@@ -664,7 +664,7 @@
     <sys:String x:Key="About_Version">버전</sys:String>
     <sys:String x:Key="About_ReleaseDate">출시일:</sys:String>
     <sys:String x:Key="About_CheckUpdate">업데이트 확인(_C)</sys:String>
-    <sys:String x:Key="About_CreatorInfo">Office Deployment Tool 기반으로 Yerong이 개발했습니다.</sys:String>
+    <sys:String x:Key="About_CreatorInfo">Office Deployment Tool 기반으로 Yerong이 개발, 한국어 번역: 비너스걸.</sys:String>
```

Comment:
Your translator info was added in `Translators` section, do not add it again here.